samplesheets/views.py

```diff
@@ -827,6 +827,28 @@ def create_app_alerts(cls, ticket, action, user):
             project=project,
         )
 
+    @classmethod
+    def create_irods_ticket(
+        cls, irods_backend, irods, path, ticket_str, expiry_date
+    ):
+        """
+        Create access ticket in iRODS.
+        :param irods_backend: IrodsBackend object
+        :param irods: iRODSSession object
+        :param path: iRODS path
+        :param ticket_str: Ticket string
+        :param expiry_date: Expiry date as datetime object
+        :raise: IrodsException
+        """
+        ticket = irods_backend.issue_ticket(
```

Comment:
I'm not exactly sure why we need this helper, can't we just call `issue_ticket()`? There doesn't really seem to be any specific functionality in this helper beyond setting the `'read'` argument.